adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -1244,6 +1244,79 @@ func CreateHealthEndpoint() *routev3.Route {
 	}
 	return &router
 }
+func CreateJwksEndpoint() *routev3.Route {
+	var (
+		router    routev3.Route
+		action    *routev3.Route_Route
+		match     *routev3.RouteMatch
+		decorator *routev3.Decorator
+	)
+
+	match = &routev3.RouteMatch{
+		PathSpecifier: &routev3.RouteMatch_Path{
```

Comment:
We add another match for GET method. That way you do not need to handle it in enforcer side.